iocage_lib/ioc_start.py

```diff
@@ -621,6 +646,22 @@ def __start_jail__(self):
                 _callback=self.callback,
                 silent=self.silent)
 
+        if manual_devfs_config:
```

Comment:
These statements are inaccurate and if included should be included at the place where what they say is happening actually happens ( which is above ).

Reply:
They absolutely should be included, otherwise it's super confusing. I noticed this myself while testing earlier after not looking at this for quite a while - only showing the dynamic ruleset id is confusing, as you don't know where it was copied from. It's the same for the default one (4) - this also confusing while debugging originally, like "what is this ruleset?".

I also wanted to make sure people understand that ruleset 4 isn't at all related to devfs_ruleset 4 on the system (they happen to server the same purpose).

I also didn't want to change the original start output (people might rely on it already somehow), otherwise I would've put it in parentheses behind it.

What about something like this?

```
 + Configured devfs to clone from ruleset {configured_devfs_ruleset}
 + Using devfs_ruleset: {devfs_ruleset}
```

and

```
 + Configured devfs to generated default ruleset
 + Using devfs_ruleset: {devfs_ruleset}
```

If changing the output is okay, we could also do

```
 + Using devfs_ruleset: {devfs_ruleset} (cloned from {configured_devfs_ruleset})
```

and

```
 + Using devfs_ruleset: {devfs_ruleset} (iocage generated)
```

Reply:
I prefer changing the output, relying on userland  output is never a good idea so users should expect things to change. It’s much cleaner than adding yet another log line and this information does need to be present.

Reply:
Done